### ISSUE TITLE
XBL: add nested binding `<xxbl:*>`

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#5568]
With #5529, we added support for `fr:keep-if-function-available`. However this is not enough as functions are used by the `xxbl:serialize-external-value` and similar attributes on `xbl:binding`.

One idea was to create child elements like:

```xml
<xxbl:serialize-external-value
    value="..."
    fr:keep-if-function-available="fr:mode"
/>
```

But see [this comment](https://github.com/orbeon/orbeon-forms/issues/5529#issuecomment-1324129845) about problems that could cause. One possibility would be to support explicitly multiple such elements with conditions in `CommonBinding`. The actual choice would be done in `AbstractBinding`.

**Comment by ebruchez** [comment]
The goal is to support, outside of Form Runner:

- `fr:date`
- `fr:number`

Consider releasing 2022.1.1 CE if needed by users.

**Comment by ricetrac** [comment]
Hello,

Do you think you can address this issue in a future version?

Thanks

**Comment by avernet** [comment]
@ricetrac `fr:date` and `fr:number` work in the latest 2024.1.2 release. Have you had a chance to try those components on that version?

**Comment by ricetrac** [comment]
Hi Alex,

No, I use only the CE version. Is there a commit I can try to integrate?

**Comment by avernet** [comment]
@ricetrac And you're saying it doesn't work out of the box for you with the 2024.1 CE release? I did a quick search through the commits made in 2025, after the 2024.1 release, and only found #6983, which was for `fr:section`.

**Comment by ricetrac** [comment]
@avernet  Yes, with 2024.1, I'm still required to add the workaround you indicated to me, as an attribute of the "<xf:model>" of pages that use a "fr:number":

xxf:assets.baseline.updates="-/ops/javascript/scalajs/orbeon-xforms-web.js
+/apps/fr/resources/scalajs/orbeon-form-runner-web.js">

**Comment by avernet** [comment]
@ricetrac Ah, I see what you mean. That's not a "workaround." Those are `fr:*` components, which are part of Form Runner, so their client-side code is only delivered through `orbeon-form-runner-web.js`. That's why you need to include that baseline update. So having it there is correct. But otherwise, the components are working fine, right?

**Comment by ricetrac** [comment]
Yes, with this information, the fr:number component works fine. But I've always considered this a workaround; I only apply it to pages that use the component.
I believe there's a drawback to doing this, but I don't remember what it is.
